11ty/src/_data/drupal.js

```diff
@@ -28,10 +28,15 @@ module.exports = async function () {
     process.exit(1);
   }
   
-  let drupalMenuData = await EleventyFetch(`https://${process.env.DRUPAL_AUTH}@design-system-cms.kubernetes.lib.umich.edu/jsonapi/menu_link_content/menu_link_content`, {
+  let drupalMenuData = await EleventyFetch(`${process.env.DRUPAL_URL}/jsonapi/menu_link_content/menu_link_content`, {
     duration: "1d", // 1 day
     type: "json",
     method: 'GET',
+    fetchOptions: {
+      headers: {
+        "Authorization": "Basic" + Buffer.from(process.env.DRUPAL_AUTH).toString('base64')
```

Comment:
@niquerio This needs to be `Basic `, with a space between `Basic` and the encoded username/password.

Reply:
Fixed! And now the menus are back.